### PR TITLE
Update the buildfarm config with the new gpg keys.

### PR DESCRIPTION
**Pull request by tfoote**
This requires https://github.com/ros/rosdistro/pull/21466 to merge.

Following up https://discourse.ros.org/t/security-issue-on-ros-build-farm/9342

### CODE REVIEW

**Comment by nuclearsandwich** [comment]
@tfoote can you justify the switch from the testing repo to the building repo for doc jobs? It's not mentioned in the commit message.

File: kinetic/source-build.yaml [hunk 1]
```diff
@@ -11,41 +11,43 @@ notifications:
   - ros-buildfarm-kinetic@googlegroups.com
   - tfoote+buildfarm@osrfoundation.org
   maintainers: true
+package_whitelist:
+  - catkin
```
Comment (nuclearsandwich) [inline kinetic/source-build.yaml]: Should this whitelist be removed?

File: index.yaml [hunk 2]
```diff
@@ -48,7 +48,7 @@ distributions:
       released-packages-without-doc-job: lunar/doc-released-build.yaml
     notification_emails:
     - ros-buildfarm-lunar@googlegroups.com
-    - mikael+buildfarm@osrfoundation.org
+    - clalancette+buildfarm@osrfoundation.org
```
Comment (nuclearsandwich) [inline index.yaml:51]: @clalancette just a heads up for the short time remaining that it will matter.

File: kinetic/release-armhf-build.yaml [hunk 3]
```diff
@@ -32,38 +32,39 @@ repositories:
...
+    fp/4W7uCPFvwAatWEHJhlM3sQNiMNStJFegr56xQu1a/cbJH7GdbseMhG/f0BaKQ
+    qXCI3ffB5y5AOLc9Hw7PYiTFQsuY1ePRhE+J9mejgWRZxkjAH/FlAubqXkDgterC
+    h+sLkzGf+my2IbsMCuc+3aeNMJ5Ej/vlXefCH/MpPWAHCqpQhe2DET/jRSaM53US
+    AHNx8kw4MPUkxExgI7Sd
+    =4Ofr
     -----END PGP PUBLIC KEY BLOCK-----
   urls:
   - http://repositories.ros.org/ubuntu/building
 target_repository: http://repositories.ros.org/ubuntu/building
+
```
Comment (nuclearsandwich) [inline kinetic/release-armhf-build.yaml]: Spurious or intentional?

File: kinetic/release-build.yaml [hunk 4]
```diff
@@ -12,39 +12,39 @@ notifications:
   - tfoote+buildfarm@osrfoundation.org
   maintainers: true
 sync:
-  package_count: 2300
+  package_count: 200
```
Comment (nuclearsandwich) [inline kinetic/release-build.yaml]: Are we ready to restore this so we don't have a testing Armageddon?

File: kinetic/release-xenial-arm64-build.yaml [hunk 5]
```diff
@@ -34,38 +34,39 @@ repositories:
...
+    fp/4W7uCPFvwAatWEHJhlM3sQNiMNStJFegr56xQu1a/cbJH7GdbseMhG/f0BaKQ
+    qXCI3ffB5y5AOLc9Hw7PYiTFQsuY1ePRhE+J9mejgWRZxkjAH/FlAubqXkDgterC
+    h+sLkzGf+my2IbsMCuc+3aeNMJ5Ej/vlXefCH/MpPWAHCqpQhe2DET/jRSaM53US
+    AHNx8kw4MPUkxExgI7Sd
+    =4Ofr
     -----END PGP PUBLIC KEY BLOCK-----
   urls:
   - http://repositories.ros.org/ubuntu/building
 target_repository: http://repositories.ros.org/ubuntu/building
+
```
Comment (nuclearsandwich) [inline kinetic/release-xenial-arm64-build.yaml]: Another added empty line.

**Comment by tfoote** [comment]
Nope, that's a copy and paste error where I copied the whole block of the repositories section with the gpg key. I noticed the realphabetization but not the repo change. Reverted.

**Review by nuclearsandwich — APPROVED** [review]

**Review by nuclearsandwich — APPROVED** [review]

**Pull request merged**